scripts/install/opt/seagate/cortx/motr/bin/motr_mini_prov.py

```diff
@@ -239,20 +302,59 @@ def update_copy_motr_config_file(self):
     dirs = [MOTR_MOD_DATA_DIR, MOTR_MOD_ADDB_STOB_DIR, MOTR_MOD_ADDB_TRACE_DIR]
     create_dirs(self, dirs)
 
-    f1 = open(f"{MOTR_SYS_CFG}", "a")
-    f1.write(f"MOTR_CONF_DIR={MOTR_CONF_DIR}\n")
-    f1.write(f"MOTR_MOD_DATA_DIR={MOTR_MOD_DATA_DIR}\n")
-    f1.write(f"MOTR_CONF_XC={MOTR_CONF_XC}\n")
-    f1.write(f"MOTR_MOD_ADDB_STOB_DIR={MOTR_MOD_ADDB_STOB_DIR}\n")
-    f1.write(f"MOTR_MOD_ADDB_TRACE_DIR={MOTR_MOD_ADDB_TRACE_DIR}\n")
-    f1.close()
+    # Update new config keys to config file /etc/sysconfig/motr
+    config_kvs = [("MOTR_CONF_DIR", f"{MOTR_CONF_DIR}"),
+                   ("MOTR_MOD_DATA_DIR", f"{MOTR_MOD_DATA_DIR}"),
```

Comment:
change MOTR_MOD_DATAT_DIR to MOTR_M0D_DATA_DIR.
it is zero not caps 'O'
 - also for MOTR_M0D_ADDB_STOB_DIR